### PR TITLE
Geth branch to master

### PR DESCRIPTION
**Pull request by mxmar**

### CODE REVIEW

File: genesisEmpty-geth.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{
+	"name": "AuthorityRound",
+
```
Comment (blazejkrzak) [inline genesisEmpty-geth.json:3]: these NL are redundant

File: setup-node-geth.sh [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+if [ `gsutil du gs://l16-common/genesis-geth.json | wc -l` -eq 1 ]
+  then
+  	#Yes, let's download it.
+  	echo "Downloading genesis"
+    gsutil cp gs://l16-common/genesis-geth.json .
+    sudo cp ./genesis-geth.json /l16/
+
+  else
+    #No, let's create it and upload to bucket.
+  	echo "All addresses have been created, proceeding to create genesis."
```
Comment (blazejkrzak) [inline setup-node-geth.sh]: tab

**Comment by blazejkrzak** [comment]
@patred20 @mxmar whats the status?

**Comment by patred20** [comment]
After refactor which combined `geth` and `parity` files into single branch, this PR became obsolete.
All changes are available in `refactor` branch with PR #10

**Pull request closed without merging**